src/constants.js

```diff
@@ -0,0 +1,15 @@
+function define(name, value) {
+  Object.defineProperty(exports, name, {
+    value: value,
+    writable: false,
+  });
+}
+
+define("SINGLE_ANS", "SINGLE");
```

Comment:
w szablonie JSONów na Trello to się nazywa CLOSED ale nazwa single jest dużo lesza więc trzeba szablon poprawić